scripts/index.js

```diff
@@ -0,0 +1,96 @@
+const state = {
+    temp: 65,
+    tempRange: "mild",
+    sky: "sunny",
+    city: "Seattle"
+};
+
+// TEMPERATURE FUNCTIONALITY//
+const tempUp = (event) => {
+    state.temp += 1;
+    const count = document.querySelector("#tempDisplay");
+    count.textContent = `${state.temp}`;
+    checkTemp();
+};
+
+const tempDown = (event) => {
+    state.temp -= 1;
+    const count = document.querySelector("#tempDisplay");
+    count.textContent = `${state.temp}`;
+    checkTemp();
+};
```

Comment:
A couple of things:

1.  You're not using the `event` parameter so you can leave it off.
2. You can combine these functions

```suggestion
const changeTemp = (increment) => {
    state.temp += increment;
    const count = document.querySelector("#tempDisplay");
    count.textContent = `${state.temp}`;
    checkTemp();
};
```

You can then use an anonymous function in the event registration like this:

```javascript
tempDownButton.addEventListener("click", () => changeTemp( -1 ) );
```